sdk/spring/spring-cloud-azure-core/src/test/java/com/azure/spring/cloud/core/factory/AbstractAzureServiceClientBuilderFactoryTests.java

```diff
@@ -105,6 +105,19 @@ void customizerShouldEffect() {
         assertEquals(2, anotherBuilder.getCustomizedTimes());
     }
 
+    @Test
+    void useConfiguration() {
+        System.setProperty(TestClientBuilder.TEST_CONFIGURATION_KEY, "custom");
+        TestClientBuilderFactory factory = new TestClientBuilderFactory(new AzureTestProperties());
+        TestClientBuilder builder = factory.build();
+        TestClient customClient = builder.build();
+        assertEquals("custom", customClient.testConfiguration);
+
+        builder.configuration(null);
+        TestClient client = builder.build();
+        assertEquals(TestClientBuilder.TEST_CONFIGURATION_DEFAULT, client.getTestConfiguration());
+    }
```

Comment:
This test case is for?

Reply:
It's used to test the configuration usage and whether the creation of `com.azure.core.util.Configuration` is normal.

Reply:
Can we update the test case name to be accurate?